docs/background/velocities.rst

```diff
@@ -0,0 +1,521 @@
+parameters of the pulsar system and the screen.
+
+
+Scintillometric observables
+===========================
+
+The basic scintillometric observable is the curvature
+:math:`\eta( t, \lambda )` of the parabola in a secondary spectrum, taken at a
+reference time :math:`t` and wavelength :math:`\lambda`. In terms of physical
+properties of the system, it can be expressed as
+
+.. math::
+
+    \eta = \frac{ \lambda^2 d_\mathrm{eff} }{ 2 c v_\mathrm{eff,\parallel}^2 },
+
+where :math:`c` is the speed of light, and :math:`d_\mathrm{eff}` and
+:math:`v_\mathrm{eff,\parallel}` are the effective distance and the effective
+velocity, respectively, to be defined below. Isolating the unknowns in this
+equation and taking the square root shows that curvature measurements constrain
+the quantity
+
+.. math::
+
+    \frac{ \lambda }{ \sqrt{ 2 c \eta } }
+        = \frac{ \left| v_\mathrm{eff,\parallel} \right| }
+               { \sqrt{ d_\mathrm{eff} } },
+
+which we will refer to as the scaled effective velocity. This also shows that
+simultaneous curvature measurements taken at different wavelengths can be
+combined, leaving a time-series of scaled effective velocities as the relevant
+scintillometric observable to be modelled or fitted.
+
+
+The effective distance
+======================
+
+The distances from Earth to the pulsar and the scattering screen are denoted by
+:math:`d_\mathrm{p}` and :math:`d_\mathrm{s}`, respectively. These two physical
+distances are combined into the effective distance
+
+.. math::
+
+    d_\mathrm{eff} = \frac{ d_\mathrm{p} d_\mathrm{s} }
+                          { d_\mathrm{p} - d_\mathrm{s} }
+                   = \frac{ 1 - s }{ s } d_\mathrm{p}.
+
+Here, :math:`s` is the fractional pulsar--screen distance
+(with :math:`0 \leq s \leq 1`), given by
+
+.. math::
+
+    s = 1 - \frac{ d_\mathrm{s} }{ d_\mathrm{p} }.
+
+.. image:: images/sketch-distances.svg
+    :align: center
+    :alt: sketch with distances
+
+
+The effective velocity
+======================
+
+The scattering of the pulsar's radiation by the screen gives rise to an
+interference pattern of bright and dark fringes in space. Because the pulsar,
+the screen, and Earth all move with respect to one another, an Earth-based
+observation will probe different points in this pattern over time.
+
+[add figure showing interference pattern in observing plane]
+
+The sky-plane velocity (i.e., the velocity component that is perpendicular
+to the direct line of sight) of the interference pattern relative to Earth,
+known as the effective velocity, is given by
+
+.. math::
+
+    \vec{v}_\mathrm{eff} = \frac{ 1 }{ s } \vec{v}_\mathrm{lens,sky}
+        - \frac{ 1 - s }{ s } \vec{v}_\mathrm{p,sky}
+        - \vec{v}_\mathrm{\oplus,sky}.
+
+Here, :math:`\vec{v}_\mathrm{p,sky}`, :math:`\vec{v}_\mathrm{lens,sky}`, and
+:math:`\vec{v}_\mathrm{\oplus,sky}` are the sky-plane velocities of the pulsar,
+the screen, and Earth, respectively, all specified relative to the Solar
```

Comment:
I'd make this "generally specified" - as one can choose any reference frame one wants.

Reply:
Added.